org-transclusion.el

```diff
@@ -942,12 +977,15 @@ based on the following arguments:
          (end-mkr)
          (ov-src (text-clone-make-overlay sbeg send sbuf)) ;; source-buffer overlay
          (tc-pair ov-src)
-         (content content))
+         (content content)
+         (at-end-of-buffer (eq (point-at-eol) (point-max))))
```

Comment:
@joshcho with reverting the commit for "removing the extra line", I don't think any of these boundary stuff is necessary. Agree?

Reply:
Yup, that sounds good.

Reply:
So for my own purposes, should I just revert that change within `org-transclusion.el`? Or is there a more elegant way of customization?